Doc/library/traceback.rst

```diff
@@ -339,7 +339,9 @@ capture data for later printing in a lightweight fashion.
       creating the :class:`StackSummary` cheaper (which may be valuable if it
       may not actually get formatted). If *capture_locals* is ``True`` the
       local variables in each :class:`FrameSummary` are captured as object
-      representations.
+      representations. If, for some reason, a representation can not be
+      created (i.e. :func:`repr` fails), it is replaced with a placeholder
+      (``'<local repr() failed>'``).
```

Comment:
It's over-specified to include the exact error string. I'm not sure we even need to write in the main text that errors are swallowed (it's not mentioned for the other cases where this is done). So I think it's not necessary as part of the main text, but we should mention the change. So how about remove this sentence, and then add below

```
   .. versionchanged:: 3.12
       Exceptions raised from :func:`repr` on a local variable are no longer propagated to the caller.
```

Reply:
> It's over-specified to include the exact error string

Can it go as "to satisfy reader's сuriosity"?

Reply:
I was unsure whether to include the exact "wording", too. I thought this way this section of the documentation pops up in the search results if users search for this error.

Reply:
We don’t normally put error msg text in the docs for the purpose of it coming up in search engines. 

We also don’t consider them part of the published API. Changing the type of an exception is a backwards compatibility break, but changing the msg is not. So that’s another indication that it shouldn’t be documented.